src/Sulu/Bundle/AutomationBundle/Admin/AutomationContentNavigationProvider.php

```diff
@@ -75,6 +76,11 @@ public function getNavigationItems(array $options = [])
         $automation->setAction('automation');
         $automation->setComponent('automation-tab@suluautomation');
         $automation->setDisplay(['edit']);
+        $automation->setDisplayConditions(
+            [
+                new DisplayCondition('url', DisplayCondition::OPERATOR_NOT_EQUAL, ''),
```

Comment:
this tab will also be used on a few other places not only pages! we should extend it in the `SuluContentBundle`